ui/src/app/zones/views/ZonesListTitle/ZonesListTitle.tsx

```diff
@@ -0,0 +1,21 @@
+import { Fragment } from "react";
+
+import { Tooltip, Icon } from "@canonical/react-components";
+
+const ZonesListTitle = (): JSX.Element => {
+  return (
+    <Fragment>
+      <span className="p-heading--four">Availability zones</span>
+      <span style={{ marginLeft: "1rem" }}>
```

Comment:
We try not to use inline styles as much as possible, but it's super common to need to move stuff around a little bit. To that end we've got a [set of utils](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/scss/_utilities.scss#L66) (`u-nudge-right`, `u-nudge-left--small`) that simply add a bit of padding.

Reply:
ahh amazing! This is what we were hoping for but didn't think they existed. Thank you!